assessment/catscores.php

```diff
@@ -86,6 +86,13 @@ function printoutcomes($arr,$ind,&$outcomenames, &$catscore, &$catposs) {
 	foreach (array_keys($catscore) as $category) {
 		if (is_numeric($category)) {
 			continue;
+		} elseif (0==strncmp($category,"AID-",4)) { //category is another assessment
+			$categoryaid=substr($category,4);
+			$query = "SELECT name FROM imas_assessments WHERE id='$categoryaid' LIMIT 1";

```

Comment:
It would probably be a good idea, just to catch any cases where the $categoryaid is, by some mistake, pointing to an assessment outside of the course.